modules/extension/xsd/xsd-sld/src/main/java/org/geotools/sld/bindings/SLDMarkBinding.java

```diff
@@ -114,6 +121,13 @@ public Object parse(ElementInstance instance, Node node, Object value) throws Ex
         Mark mark = styleFactory.createMark();
 
         if (wkName != null) {
+            // allow references to files containing mark definitions
+            if (wkName.startsWith("file://")) {
+                URL url = resourceLocator.locateResource(wkName);
+                if (url != null) {
```

Comment:
Would an warning message be appropriate where in the case the SVG is not found ?

Reply:
Good idea, adding logging